backend/src/main/java/fk/prof/backend/model/association/impl/ZookeeperBasedBackendAssociationStore.java

```diff
@@ -396,13 +405,15 @@ private void loadDataFromZookeeperInBackendLookup() throws Exception {
 
     List<String> backendZNodeNames = curatorClient.getChildren().forPath(backendAssociationPath);
     for(String backendZNodeName: backendZNodeNames) {
+      logger.debug("Found associations for : {}", backendZNodeName);
       String backendZNodePath = getZNodePathForBackend(backendZNodeName);
       List<String> processGroupNodes = curatorClient.getChildren().forPath(backendZNodePath);
       Set<Recorder.ProcessGroup> processGroups = new HashSet<>();
 
       for(String processGroupZNodeName: processGroupNodes) {
         String processGroupZNodePath = getZNodePathForProcessGroup(backendZNodeName, processGroupZNodeName);
         Recorder.ProcessGroup processGroup = Recorder.ProcessGroup.parseFrom(ZookeeperUtil.readZNode(curatorClient, processGroupZNodePath));
+        logger.debug("\t{},{},{}", processGroup.getAppId(), processGroup.getCluster(), processGroup.getProcName());
```

Comment:
Add isDebugEnabled check. Bracket notation helps avoid concatenation if not necessary but arg evaluation still happens. This is a one-time log so not super-critical but generally a good practice to have the check.